nextflow/vcf_prepper/nextflow.config

```
@@ -89,11 +98,35 @@ process {
 }
 
 executor {
-  queueSize = 1200
+  queueSize = params.queue_size
   submitRateLimit = '50/10sec'
 }
 
 singularity {
   enabled = true
   autoMounts = true
 }
+
+trace {
+    enabled = true
```

If it is helpful, we can also mention the [fields](https://www.ebi.ac.uk/seqdb/confluence/display/ENS/Ensembl+Coding+Guidelines#heading-Recommendations) we want in trace which also include % of CPU or memory used per process

```suggestion
  trace {
    // enabled = true
    fields = 'task_id,name,process,hash,status,exit,attempt,submit,time,realtime,cpus,%cpu,memory,%mem'
    // filter = 'name =~ /foo.*/ && status == "FAILED"'
}
```